RPLCD/gpio.py

```diff
@@ -218,3 +235,9 @@ def _pulse_enable(self):
         c.usleep(1)
         GPIO.output(self.pins.e, 0)
         c.usleep(100)  # commands need > 37us to settle
+
+    def _wait(self):
+        """Rate limit the number of send events."""
+        end = self.last_send_event + (700.0 / 1000000.0)
```

Comment:
This value (the delay) could be extracted into a constant (e.g. `LEGACY_MODE_WAIT_TIME`).

Reply:
Good call. I've made that change.